try:
    a = 340/3

except Exception as e:
    print(e)


# Execute when there is no error.
else:
    print("Hey it is working fine!")